src/screens/Library/components/GameCard/index.tsx

```diff
@@ -141,11 +142,9 @@ const GameCard = ({
     }
     if (hasUpdate) {
       return (
-        <FontAwesomeIcon
-          size={'2x'}
-          icon={faRepeat}
-          onClick={() => handleUpdate()}
-        />
+        <SvgButton onClick={() => handleUpdate()}>
+          <FontAwesomeIcon size={'2x'} icon={faRepeat} />
```

Comment:
When I tested this branch, for some reason this icon is Black instead of green. I think some class is overriding it.
so try to make it use the correct color.

Reply:
fixed this, thanks